google/cloud/storage/oauth2/google_credentials_test.cc

```diff
@@ -55,23 +87,37 @@ TEST_F(CredentialsTest, Insecure) {
  * requires valid keys and contacting Google's production servers, and would
  * make this an integration test.
  */
-TEST_F(CredentialsTest, LoadValidAuthorizedUserCredentials) {
+TEST_F(GoogleCredentialsTest, LoadValidAuthorizedUserCredentials) {
   char const filename[] = "authorized-user.json";
   std::ofstream os(filename);
-  os << R"""({
+  std::string contents_str = R"""({
```

Comment:
If you are doing this, move the definition of `os` closer to where it is first used .